src/skins/vector/css/common.css

```diff
@@ -264,6 +250,12 @@ input[type=text]:focus, textarea:focus {
     color: white;
 }
 
+.emojione {
```

Comment:
weird, i must have pushed that to the wrong branch - or someone backed out my change. Filed separately as https://github.com/vector-im/vector-web/issues/1934